### ISSUE TITLE
virtualenv package with dot missing

### ISSUE THREAD

**Issue by gjcarneiro** [indygreg/PyOxidizer#60]
I am trying to package something that has a virtualenv package with a dot

```
(env) 05:40:29 (master) ~/projects/yacron$ pip freeze|grep ruamel
ruamel.yaml==0.15.97
```
This specific package seems to be skipped by pyoxidizer:

```
Traceback (most recent call last):
  File "yacron.__main__", line 8, in <module>
  File "yacron.cron", line 8, in <module>
  File "yacron.config", line 6, in <module>
  File "strictyaml", line 2, in <module>
  File "strictyaml.parser", line 5, in <module>
ModuleNotFoundError: No module named 'ruamel'
SystemError
```

Extract of the config:

```
# Package things from a populated virtualenv.
[[packaging_rule]]
type = "virtualenv"
path = "/home/gjc/projects/yacron/env"

[[embedded_python_run]]
mode = "module"
module = "yacron.__main__"

[[pyoxidizer]]
version = "0.1.3"
commit = ""
```

**Comment by indygreg** [comment]
There are some changes in the `0.2` release that should help with this. And if I install `ruamel.yaml` via a `pip-install-simple` rule, this package appears to work:

```
processing packaging rule: PipInstallSimple(PackagingPipInstallSimple { package: "ruamel.yaml", optimize_level: 0, excludes: [], include_source: true, install_location: Embedded, extra_args: None })
installing modified distutils to /tmp/pyoxidizer-pip-install.6PFLys2AWWLZ/packages
modifying distutils/_msvccompiler.py for oxidation
modifying distutils/command/build_ext.py for oxidation
modifying distutils/unixccompiler.py for oxidation
pip installing to /tmp/pyoxidizer-pip-install.6PFLys2AWWLZ/install
Collecting ruamel.yaml
  Downloading https://files.pythonhosted.org/packages/52/f4/7cf7a8bb5d0ddb4311d6503caa3a7505deada47b346b696ddf1aabbb9b3b/ruamel.yaml-0.15.97.tar.gz (427kB)
Installing collected packages: ruamel.yaml
  Running setup.py install for ruamel.yaml: started
    Running setup.py install for ruamel.yaml: finished with status 'done'
Successfully installed ruamel.yaml-0.15.97
adding embedded module source: ruamel.yaml
adding embedded module bytecode: ruamel.yaml
adding embedded module source: ruamel.yaml.anchor
adding embedded module bytecode: ruamel.yaml.anchor
adding embedded module source: ruamel.yaml.comments
adding embedded module bytecode: ruamel.yaml.comments
adding embedded module source: ruamel.yaml.compat
adding embedded module bytecode: ruamel.yaml.compat
adding embedded module source: ruamel.yaml.composer
adding embedded module bytecode: ruamel.yaml.composer
adding embedded module source: ruamel.yaml.configobjwalker
adding embedded module bytecode: ruamel.yaml.configobjwalker
adding embedded module source: ruamel.yaml.constructor
adding embedded module bytecode: ruamel.yaml.constructor
adding embedded module source: ruamel.yaml.cyaml
adding embedded module bytecode: ruamel.yaml.cyaml
adding embedded module source: ruamel.yaml.dumper
adding embedded module bytecode: ruamel.yaml.dumper
adding embedded module source: ruamel.yaml.emitter
adding embedded module bytecode: ruamel.yaml.emitter
adding embedded module source: ruamel.yaml.error
adding embedded module bytecode: ruamel.yaml.error
adding embedded module source: ruamel.yaml.events
adding embedded module bytecode: ruamel.yaml.events
adding embedded module source: ruamel.yaml.loader
adding embedded module bytecode: ruamel.yaml.loader
adding embedded module source: ruamel.yaml.main
adding embedded module bytecode: ruamel.yaml.main
adding embedded module source: ruamel.yaml.nodes
adding embedded module bytecode: ruamel.yaml.nodes
adding embedded module source: ruamel.yaml.parser
adding embedded module bytecode: ruamel.yaml.parser
adding embedded module source: ruamel.yaml.reader
adding embedded module bytecode: ruamel.yaml.reader
adding embedded module source: ruamel.yaml.representer
adding embedded module bytecode: ruamel.yaml.representer
adding embedded module source: ruamel.yaml.resolver
adding embedded module bytecode: ruamel.yaml.resolver
adding embedded module source: ruamel.yaml.scalarbool
adding embedded module bytecode: ruamel.yaml.scalarbool
adding embedded module source: ruamel.yaml.scalarfloat
adding embedded module bytecode: ruamel.yaml.scalarfloat
adding embedded module source: ruamel.yaml.scalarint
adding embedded module bytecode: ruamel.yaml.scalarint
adding embedded module source: ruamel.yaml.scalarstring
adding embedded module bytecode: ruamel.yaml.scalarstring
adding embedded module source: ruamel.yaml.scanner
adding embedded module bytecode: ruamel.yaml.scanner
adding embedded module source: ruamel.yaml.serializer
adding embedded module bytecode: ruamel.yaml.serializer
adding embedded module source: ruamel.yaml.timestamp
adding embedded module bytecode: ruamel.yaml.timestamp
adding embedded module source: ruamel.yaml.tokens
adding embedded module bytecode: ruamel.yaml.tokens
adding embedded module source: ruamel.yaml.util
adding embedded module bytecode: ruamel.yaml.util
adding embedded resource: PKG-INFO
adding embedded resource: SOURCES.txt
adding embedded resource: dependency_links.txt
adding embedded resource: installed-files.txt
adding embedded resource: namespace_packages.txt
adding embedded resource: requires.txt
adding embedded resource: top_level.txt
adding embedded built extension module: _ruamel_yaml
adding empty module for missing package ruamel
```

And at run-time:

```
>>> import ruamel.yaml
>>>
```

Since I am able to import this package in my builds and since I think this bug was fixed today in the `0.2` release, I'm going to call this issue closed. If you can still reproduce in `0.2`, please reopen.

**Comment by gjcarneiro** [comment]
Confirmed to be fixed, thanks!

(got a different problem next)
